docs/content/overview/cdc-pipeline.md

```diff
@@ -11,6 +11,22 @@ CDC Streaming ELT Framework is a stream data integration framework that aims to
 * ✅ Synchronization of entire databases 
 * ✅ Schema evolution capability
 
+
+## Supported Connectors
+
+| Connector                                                | Database                                                                                                                                                                                                                                                                                                                                                                                               | Driver                   |
+|----------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|
+| [doris-pipeline](../pipelines/doris-pipeline.md)         | <li> [Doris](https://doris.apache.org/): 1.x, 2.x                                                                                                                                                                                                                                                                                                                                                  | Doris Driver: 1.5.0      |
```

Comment:
Maybe we can specify the start version to 1.2, as Light table structure changes was intruduced in [1.2](https://doris.apache.org/docs/1.2/releasenotes/release-1.2.0) release.

Reply:
OK, I've fixed this